client/components/feature-modules/blocks/components/forms/widgets/dropdown-widget.tsx

```
@@ -0,0 +1,90 @@
+
+    return (
+        <div className="space-y-2">
+            <Label htmlFor={label} className={cn(hasErrors && "text-destructive")}>
+                {label}
+            </Label>
+            {description && <p className="text-sm text-muted-foreground">{description}</p>}
+            <Popover open={open} onOpenChange={setOpen}>
+                <PopoverTrigger asChild>
+                    <Button
+                        variant="outline"
+                        role="combobox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)